sdk/azidentity/aad_identity_client.go

```diff
@@ -192,7 +192,7 @@ func (c *aadIdentityClient) createRefreshTokenRequest(ctx context.Context, tenan
 	data.Set(qpScope, strings.Join(scopes, " "))
 	dataEncoded := data.Encode()
 	body := azcore.NopCloser(strings.NewReader(dataEncoded))
-	req, err := azcore.NewRequest(ctx, http.MethodPost, azcore.JoinPaths(c.authorityHost, tenantID, tokenEndpoint))
+	req, err := azcore.NewRequest(ctx, http.MethodPost, azcore.JoinPaths(c.authorityHost, tenantID, getIdentityEndpointSuffix(tenantID)))
```

Comment:
Do we need to provide a way for callers to specify a complete URL here (stack scenario)?

Reply:
Not as far as I know yet. They will change the authority host, pass in the "adfs" tenant ID and we will automatically add the correct ADFS endpoint for them. Will get back to you after testing this against an ADFS instance in case something needs to change, but it seems this is how it works.

Reply:
I dont think there is a case were a fully custom URL needs to be passed in.